### DISCUSSION TITLE
Change Preset Profile contact / social background color

### DISCUSSION THREAD

**Discussion by cloudguy23** [templaza/astroid-framework#290] [General]
Hi Everyone,

I'm upgrading my Asteroid template and unable figure out how to officially change the background, text, link color of the top-most contact / social bar.   It defaults to whatever Profile Is set - e.g. cherry, raspberry, etc.   I want mine to be black text on white background (attached screenshot, area to change indicated with black arrow).  I'm using v2.6.3 for both framework and template.

<img width="366" alt="Screenshot 2023-02-11 at 14 06 25" src="https://user-images.githubusercontent.com/26149515/218276589-6600752c-038f-4a6a-8a49-cf37179d5d67.png">

Overriding backup / text colors in Layout->Topbar doesn't seem to do the trick for the topbar section specifically.   Works for other sections.

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/26149515/218276962-6b255915-420f-45a0-9007-6317adccdfb4.png">

Thank you, appreciate any insights.

**Comment by Chacapamac** [comment]
I @cloudguy23 
I guess you are using Template Zero
You have problem changing the Topbar background color because it is set in the **Topbar**class with the “**bg-primary”** class that will show whatever your chosen layout **Theme Primary** color

![bg-primary](https://user-images.githubusercontent.com/1907276/218483225-c8dd529f-5f88-48af-af3c-5e6ec87afa17.jpg)

I didn’t know but I guess it is probably a class system that you can use in Astroid base on your **Theme colors** — I think its related to Bootstrap Theming???
![Screen Shot 2023-02-13 at 9 19 58 AM](https://user-images.githubusercontent.com/1907276/218483926-50f03bee-2519-4e82-93c0-72c993f1de2f.jpg)

**Logically you should be able to use:**
bg-primary (for background) or primary (for color)
bg-secondary (for background) or secondary (for color) 
bg-success (for background) or success (for color)   

**I’m not sure about that use, maybe @sonvnn  or another user can confirm?**
**That should go in the documentation**

When you remove the class you can control the background and text/link color in the “**Topbar**” section or in the “**Topbar-1 or 2**” subsections for more control if needed.

**Reply by cloudguy23** [reply]
Thank you @sonvnn ..  Missed the obvious class.   I removed the custom class and was able to Topbar text / background using the Design Settings Tab.

**Reply by Chacapamac** [reply]
Great @cloudguy23 
Can we put that conversation as **Solved**?

Good Day!

**Reply by cloudguy23** [reply]
Please mark issue as solved.   Thank you.

**Reply by Chacapamac** [reply]
Good Day to the Astroid Community!
